### DISCUSSION TITLE
How to handle different environments and CI/CD

### DISCUSSION THREAD

**Discussion by stephen776** [graphql-editor/graphql-zeus#283] [Q&A]
Is it recommended to run the cli generation as part of a CI build step? (After using local for dev)

**Comment by seanonthenet** [comment]
Hey @stephen776 
What is your setup like? I wouldn't think you'd need to do it. I am committing generated files to version control.

**Reply by stephen776** [reply]
I'm still in the planning stages of the build pipeline. I think committing the generated files makes sense as well so I'll probably go that route

**Reply by seanonthenet** [reply]
Nice. 👍

**Comment by aexol** [comment]
If you want to be 100% sure that schema did not change during build time, you can add `zeus` folder to `.gitignore` and add the generation command  after `npm install` and before `npm run build`
